src/goto-harness/recursive_initialization.cpp

```diff
@@ -85,6 +86,22 @@ bool recursive_initialization_configt::handle_option(
       [](const std::string &opt) -> irep_idt { return irep_idt{opt}; });
     return true;
   }
+  else if(option == COMMON_HARNESS_GENERATOR_MEMBER_SELECTION_OPT)
```

Comment:
Correct me if I’m wrong, but this looks as if you can only have one member havoc'd at a time? Because that shouldn’t be the case.

Reply:
Done.